### PR TITLE
added _links property support in projects api

### PR DESCRIPTION
**Pull request by etsrepo**
`_links` added projects API, below is the example for API response

`
 "_links": {
        "self": "http://gitlab.com/api/v4/projects/153",
        "issues": "http://gitlab.com/api/v4/projects/153/issues",
        "merge_requests": "http://gitlab.com/api/v4/projects/153/merge_requests",
        "repo_branches": "http://gitlab.com/api/v4/projects/153/repository/branches",
        "labels": "http://gitlab.com/api/v4/projects/153/labels",
        "events": "http://gitlab.com/api/v4/projects/153/events",
        "members": "http://gitlab.com/api/v4/projects/153/members"
    },
`

### CODE REVIEW

File: projects.go [hunk 1]
```diff
@@ -158,6 +159,16 @@ type ForkParent struct {
 	WebURL            string `json:"web_url"`
 }
 
+type Links struct {
```
Comment (svanharmelen) [inline projects.go]: Could you add a comment here to satisfy `golint`?

File: projects.go [hunk 2]
```diff
@@ -63,6 +63,7 @@ type Project struct {
 	ContainerRegistryEnabled                  bool              `json:"container_registry_enabled"`
 	CreatedAt                                 *time.Time        `json:"created_at,omitempty"`
 	LastActivityAt                            *time.Time        `json:"last_activity_at,omitempty"`
+	Links                                     *Links            `json:"_links,omitempty"`
```
Comment (svanharmelen) [inline projects.go]: Please place this below `Statistics` as I try to keep the order of these fields inline with the order used in the docs.

Comment (etsrepo) [reply projects.go]: sure

**Comment by etsrepo** [comment]
updated with comments

**Comment by svanharmelen** [comment]
Thanks @etsrepo! LGTM!

**Pull request merged**